### PR TITLE
fs-2128-scoring-unscored-sub-crits

### PR DESCRIPTION
**Pull request by gio-karunakaran**

### CODE REVIEW

File: app/assess/routes.py [hunk 1]
```diff
@@ -144,6 +144,32 @@ def display_sub_criteria(
...
+
+            return redirect(
+                url_for(
+                    "assess_bp.display_sub_criteria",
+                    application_id=application_id,
+                    sub_criteria_id=sub_criteria_id,
+                    theme_id=theme_id,
+                )
+            )
+
```
Comment (harryyo) [inline app/assess/routes.py:172]: I think this is already handled now on line 45-65, No?

**Review by harryyo — CHANGES_REQUESTED** [review]
One bit I think maybe can be removed...?

File: app/assess/routes.py (hunk 1 above)
Comment (gio-karunakaran) [reply app/assess/routes.py]: you're right! have sorted this now 👍🏾

**Review by ramsharma-prog — CHANGES_REQUESTED** [review]

File: app/assess/routes.py [hunk 2]
```diff
@@ -144,6 +144,32 @@ def display_sub_criteria(
             (1, "Poor"),
         ]
 
+        if request.args.get("add-comment") == "1":
+            display_comment_box = True
```
[the remark this answers is not in the document]
Comment (gio-karunakaran) [reply app/assess/routes.py]: sorted!

**Review by harryyo — APPROVED** [review]
👍

**Review by ramsharma-prog — APPROVED** [review]
LGTM 😄

**Pull request merged**